trivy-secret.yaml

```diff
@@ -83,6 +83,34 @@ rules:
     regex: UID2-O-P-[0-9]*-(?P<secret>.{6}\..{38})
     secret-group-name: secret
 
+  ########################################
+  # UID2 Client Side Keypair Private Key #
+  ########################################
+  - id: uid2-client-side-keypair-private-key-test
+    category: uid2
+    title: UID2 - Client Side Keypair Private Key - Test
+    severity: CRITICAL
+    keywords:
+      - UID2-Y-T
+    regex: UID2-Y-T-[0-9]*-(?P<secret>.{92})
```

Comment:
Why the `-[0-9]*`? It's not in https://github.com/IABTechLab/uid2-optout/pull/42/files

Reply:
Some keys in admin are prefixed like UID2-C-T-999-secret, so the other keys in trivy-secret.yaml are using this regex pattern. Im am not sure why.

Reply:
Ah ok, it looks like that number is the site ID. Keypairs don't have the site ID in their names.